src/main/java/com/es/phoneshop/model/order/Order.java

```diff
@@ -0,0 +1,37 @@
+package com.es.phoneshop.model.order;
+
+import com.es.phoneshop.model.cart.Cart;
+import com.es.phoneshop.model.contactDetails.ContactDetails;
+import com.es.phoneshop.model.deliveryDetails.DeliveryDetails;
+import com.es.phoneshop.model.entity.Entity;
+import com.es.phoneshop.model.idGenerator.UUIDGenerator;
+import com.es.phoneshop.model.paymentMethod.PaymentMethod;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.math.BigDecimal;
+
+@Getter
+@Setter
+public class Order extends Cart implements Entity {
+
+    private Long id;
+    private String secureId;
```

Comment:
why  do we need 2 id-s

Reply:
Overview page displays private information (phone number, adress, surname ...), so when we use long in overview url it is pretty easy to guess some valid overview id and check private info, with String id it is less likely, because it is much harder to guess

Reply:
Not sure abiut "easy to guess the valid id"
But anyway then we can use one secruId everywhere, no need to ovecomplicate model with many unique fileds